Add the FortranCon2020 community abstract (#3)
fortran-lang/talks: FortranCon2020-community/README.md

**Abstract**:
A thriving community around a programming language is essential for onboarding
newcomers to the language and retaining existing users.
Besides a few discussion boards and mailing lists, Fortran has not had a
healthy online community in the modern internet era, like many other
programming languages have.
Despite the developments of the language itself, its user base has been
declining as a result.
Many new software projects are now being started in languages like C++, Python,
or Julia, despite Fortran being better suited for the task at hand.
In this talk, we will discuss the need for and the ongoing development of the new
fortran-lang (https://fortran-lang.org) community and its suite of projects
that currently include a standard library, package manager, and a website.
Fortran-lang aims to provide a central place for Fortran users, beginning and
expert alike, to find a set of recommended tools, libraries, tutorials, and
discussion venues that are inclusive and welcoming of newcomers to the language.
We believe that an organized community and a home of Fortran on the internet,
in addition to the ongoing work by the Standards Committee and compiler developers,

> are essential for the long-term thriving--and not just mere survival--of Fortran.

Reviewer: I think this is easier to read:
```suggestion
are essential for the long-term thriving -- and not just mere survival -- of Fortran.
```

Author: Okay, I disagree but I want to move on. It's not a deal breaker.

Reviewer: I didn't know you disagreed, I thought it was perhaps a mistake. I am fine either way, you are the presenter, so I am fine with whatever way you prefer (feel free to revert the change).

newcomers to the language and retaining existing users.
Besides a few discussion boards and mailing lists, Fortran has not had a
healthy online community in the modern internet era, like many other
programming languages have.
Despite the developments of the language itself, its user base has been
declining as a result.
Many new software projects are now being started in languages like C++, Python,
or Julia, despite Fortran being better suited for the task at hand.
In this talk, we will discuss the need for and the ongoing development of the new
fortran-lang (https://fortran-lang.org) community and its suite of projects
that currently include a standard library, package manager, and a website.
Fortran-lang aims to provide a central place for Fortran users, beginning and
expert alike, to find a set of recommended tools, libraries, tutorials, and
discussion venues that are inclusive and welcoming of newcomers to the language.
We believe that an organized community and a home of Fortran on the internet,
in addition to the ongoing work by the Standards Committee and compiler developers,
are essential for the long-term thriving -- and not just mere survival -- of Fortran.

> Finally, we'll discuss the next steps, the one-year vision, and the 10-year

Reviewer: I think one should use expanded forms:
```suggestion
Finally, we will discuss the next steps, the one-year vision, and the 10-year
```

**Title**: Toward a thriving open source Fortran community

**Abstract**:
A thriving community around a programming language is essential for onboarding
newcomers to the language and retaining existing users.
Besides a few discussion boards and mailing lists, Fortran has not had a 
healthy online community in the modern internet era, like many other 
programming languages have.
Despite the developments of the language itself, its user base has been 

> declining as the result. 

Reviewer 2: ```suggestion
declining as a result. 
```

Many new software projects are now being started in languages like C++, Python,
or Julia, despite Fortran being better suited for the task at hand.

> In this talk, we'll discuss the need for and the development of the new 

Reviewer 3: ```suggestion
In this talk, we'll discuss the need for and introduce the ongoing development of the new 
```
Just to highlight that this project is already started.

Author: @jvdp1 good point. However I think your current suggestion breaks the English language. I think it should be:

```suggestion
In this talk, we'll discuss the need for and the ongoing development of the new
```

> In this talk, we'll discuss the need for and the ongoing development of the new

Reviewer: ```suggestion
In this talk, we will discuss the need for and the ongoing development of the new
```
I think it's better to use the expanded form in writing.

Author: It's a matter of style and I prefer contractions where convenient as they allow for a personal and truer to self voice. I accept this suggestion here because this is community work.

fortran-lang (https://fortran-lang.org) community and its suite of projects.
Fortran-lang aims to provide a central place for Fortran users, beginning and 
expert alike, to find a set of recommended tools, libraries, tutorials, and 
discussion venues.
We believe that an organized online community, besides the ongoing work by the
Standards Committee and compiler developers, is essential for the long-term 

> thriving--and not just mere survival--of Fortran.

Reviewer: ```suggestion
thriving -- and not just mere survival -- of Fortran.
```

Author: Most style guides recommend no spaces around em dash, unless writing for a newspaper. I'll keep no spaces.
